sample/src/main/java/jp/pokepay/pokepay/LowLevelAPITests.java

```diff
@@ -371,7 +369,7 @@ public String CpmTest() throws BankRequestError, ProcessingError {
         System.out.println("客のアクセストークンでCpmをcreate");
         CreateAccount createAccount = new CreateAccount("accountTest", pokepayMoneyId);
         Account account = createAccount.send(customerAccessToken);
-        CreateAccountCpmToken createAccountCpmToken = new CreateAccountCpmToken(account.id, CreateAccountCpmToken.SCOPE_BOTH, 100, "data");
+        CreateAccountCpmToken createAccountCpmToken = new CreateAccountCpmToken(account.id, CreateAccountCpmToken.SCOPE_BOTH, 100, new Metadata("value1"));
```

Comment:
It will be converter to json string before it requests.

[converter to json string](https://github.com/pokepay/android-sdk/blob/f0ae38b0b6a21383f1493eba26cf5e25625d8cf2/pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/Account/CreateAccountCpmToken.java#L59)
[do this before it be sent](https://github.com/pokepay/android-sdk/blob/f0ae38b0b6a21383f1493eba26cf5e25625d8cf2/pokepaylib/src/main/java/jp/pokepay/pokepaylib/BankAPI/Account/CreateAccountCpmToken.java#L51)